### ISSUE TITLE
TAC Election: Electorate

### ISSUE THREAD

**Issue by rhaning** [ossf/tac#66]
- Those wishing to vote in the election will request a ballot by submitting a (to be created) Google Form.
- Election Officials will determine eligibility based upon participation/contributions. Note that a contribution does not specifically imply only code contributions.

**Comment by jenniferfernick** [comment]
To add some details behind this thinking (@rhaning and others, please comment/clarify if I misstate anything): The intention here is that the Google Form will be used to give those wishing to vote in the TAC election an opportunity to note any of their contributions to OpenSSF this year, as we will use a broad definition of "OpenSSF contributors" to define the population of people eligible to vote in the TAC election. 

Importantly:

- Contributions can take many forms and need not be code and thus need not necessarily the types of things that could be easily determined through scraping a GitHub repo 

- Self-identification seems like the fairest way for people to indicate contributions and thus eligibility

- The intent is to announce publicly (on the OpenSSF-announcements mailing list, and possibly others as well) the election procedures and deadlines, including for self-identifying as an eligible voter in the TAC election

**Comment by rhaning** [comment]
@jenniferfernick That is correct, and thank you for detailing it much more effectively than I did. :)

**Comment by dlorenc** [comment]
+1, I agree here.

**Comment by AevaOnline** [comment]
+1, thanks for the great summary Ryan and Jennifer!

**Comment by estesp** [comment]
+1

**Comment by dlorenc** [comment]
Closing as the election is finished!
